### PR TITLE
Fix typo and some comments on uppercase variables in pkg/pod.

### PR DESCRIPTION
**Pull request by cccfeng**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

# Release Notes

```
Describe any user facing changes here, or delete this block.

Examples of user facing changes:
- API changes
- Bug fixes
- Any changes in behavior

```

### CODE REVIEW

File: pkg/pod/resource_request.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func resolveResourceRequests(containers []corev1.Container) []corev1.Container {
 		}
 	}
 
-	// Set resource requests for all steps but the theh last container to
+	// Set resource requests for all steps but the the last container to
```
Comment (sbwsg) [inline pkg/pod/resource_request.go]: Ah, correct spelling but we actually don't need the second `the` - this should become `but the last container`

Comment (cccfeng) [reply pkg/pod/resource_request.go]: I think I'm dazzled and need a rest... I force push to my branch for fixing it.

**Pull request merged**